### PR TITLE
Documentation updates.

### PR DESCRIPTION
**Pull request by FreddieWitherden**

### CODE REVIEW

File: doc/src/user_guide.rst [hunk 1]
```diff
@@ -1264,17 +1300,38 @@ Time average quantities. Parameterised with
...
+    ``single`` | ``double``
+
+7. ``region`` --- region to be averaged, specified as either the
+   entire domain using ``*``, a cuboidal sub-region via diametrically
+   opposite vertices, or a sub-region of elements that have faces on a
+   specific domain boundary via the name of the domain boundary
+
+    ``*`` | ``[(x, y, [z]), (x, y, [z])]`` | *string*
+
+8. ``avg-name`` --- expression to time average, written as a function of
```
Comment (pv101) [inline doc/src/user_guide.rst:1327]: ``avg-name`` -> ``avg-*name*``

File: doc/src/user_guide.rst [hunk 2]
```diff
@@ -1264,17 +1300,38 @@ Time average quantities. Parameterised with
...
+   the primitive variables and gradients thereof; multiple expressions,
+   each with their own *name*, may be specified:
 
     *string*
 
-5. ``avg-name`` --- expression to time average, written as a function of
-   the primitive variables, time (t), and space (x, y, [z]); multiple
+9. ``fun-avg-name`` --- expression to compute at file output time,
```
Comment (pv101) [inline doc/src/user_guide.rst:1333]: ``fun-avg-name`` -> ``fun-avg-*name*``

File: doc/src/user_guide.rst [hunk 3]
```diff
@@ -1289,6 +1346,7 @@ Example::
 
     avg-p = p
     avg-p2 = p*p
+    fun-avg-varp = p2 - p*p
```
Comment (pv101) [inline doc/src/user_guide.rst:1349]: So, just checking, the p here is the averaged p from avg-p, not the instantaneous p?

File: doc/src/user_guide.rst [hunk 4]
```diff
@@ -72,13 +80,6 @@ have one of the following partitioners installed:
 1. `metis <http://glaros.dtc.umn.edu/gkhome/views/metis>`_ >= 5.0
 2. `scotch <http://www.labri.fr/perso/pelegrin/scotch/>`_ >= 6.0
 
-Importing CGNS Meshes
```
Comment (pv101) [inline doc/src/user_guide.rst:75]: Can you also remove reference to CGNS around L190 below.

File: doc/src/user_guide.rst (hunk 3 above)
Comment (FreddieWitherden) [reply doc/src/user_guide.rst:1349]: Yes, instantaneous values here would not make sense and prefixing with avg- is confusing.

Comment (pv101) [reply doc/src/user_guide.rst:1349]: Agreed.

**Comment by FreddieWitherden** [comment]
Fixed.

**Pull request merged**